src/uct/ib/base/ib_device.c

```diff
@@ -1475,6 +1513,58 @@ uct_ib_device_get_roce_ndev_name(uct_ib_device_t *dev, uint8_t port_num,
     return UCS_OK;
 }
 
+ucs_status_t
+uct_ib_device_get_roce_ndev_index(uct_ib_device_t *dev, uint8_t port_num,
+                                  int gid_index, int *ndev_index_p)
```

Comment:
pls use uint8_t for the gid index in all places, as it's used in verbs.h structs